packages/webview_flutter/webview_flutter_android/test/instance_manager_test.dart

```diff
@@ -7,29 +7,137 @@ import 'package:webview_flutter_android/src/instance_manager.dart';
 
```

Comment:
Copy of: https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_wkwebview/test/src/common/instance_manager_test.dart